go.mod

```diff
@@ -4,7 +4,7 @@ go 1.14
 
 require (
 	github.com/envoyproxy/go-control-plane v0.9.4
-	github.com/envoyproxy/protoc-gen-validate v0.3.0
+	github.com/envoyproxy/protoc-gen-validate v0.1.0
```

Comment:
why are we doing this?

Reply:
I can swear we used to have a valid 0.3.0. :/

Reply:
That's not a real version afaict

![image](https://user-images.githubusercontent.com/3442374/77348604-f9fbba00-6cf6-11ea-8641-dcfe0dcaafd5.png)

`go: github.com/envoyproxy/protoc-gen-validate@v0.3.0: unknown revision v0.3.0`

Reply:
Chatted offline, I'll revert this. Unclear where the tag is being pulled from but seems to work for Go 1.14. I was running a older version of Go.